experimental/language-generation/README.md

```diff
@@ -93,23 +82,22 @@ Your bot might target more than one spoken/display language. To do this, you can
 
 The current library does not include any capabilities for grammar check or correction.
```

Comment:
Either need more less, i.e. drop or say we want feedback or its on the roadmap...

Reply:
Stating this here so it is clear and on top. Could refine this messaging in the official docs.